tooling/stylelint/custom-property-no-foundation-color/index.js

```diff
@@ -81,6 +81,7 @@ const allAllowedKeys = [
   "animation",
   "typography",
   "size",
+  "opacity",
```

Comment:
we can have this for now; other PR removes all direct usages so can take out again then